### DISCUSSION TITLE
loading修改

### DISCUSSION THREAD

**Discussion by ratelgogo** [umijs/umi#7059] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/23555154/126754576-4b563dfe-8ef4-4ece-88a6-311f6e63a6be.png)
我是后端，用得ant design pro ，组件切换的时候有一个loading的过渡，想问下这个如何修改，有点丑，想用spin代替

**Comment by Does666** [comment]
同问

**Reply by renkunx** [reply]
dynamicImport 里可以定义loading组件，详细查看配置文档

**Comment by ratelgogo** [comment, marked as answer]
`
config.ts中 dynamicImport -> loading 配置
`
![image](https://user-images.githubusercontent.com/23555154/143180636-0a0931e4-dc22-4326-91f3-0583ef857d7a.png)

**Reply by penny006** [reply]
为啥我配了还是没变呢？？
